options/locale/locale_en-US.ini

```diff
@@ -1436,6 +1436,7 @@ issues.next = Next
 issues.open_title = Open
 issues.closed_title = Closed
 issues.draft_title = Draft
+issues.num_comments_1 = %d comment
 issues.num_comments = %d comments
```

Comment:
we should not change this to `issues.num_comments_n` ?

like here https://github.com/go-gitea/gitea/blob/2a78c2d1f0a950c594c522e66b6e77d4616f13b5/options/locale/locale_en-US.ini#L1699-L1707

Reply:
A key rename would invalidate all existing translations, but it would be the right thing to do of course. Ideally we would have a way to automatically re-validate on Crowdin.